### ISSUE TITLE
DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead.

### ISSUE THREAD

**Issue by thecotne** [feross/eslint-config-standard-react#48]
```
(node:36737) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard-react")
(node:36737) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "node_modules/eslint-config-standard-jsx/index.js")
(node:36737) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
```

**Comment by shoota** [comment]
FYI. relational issue.
https://github.com/standard/eslint-config-standard/issues/114

**Comment by lfender6445** [comment]
The fix for me was to downgrade eslint from 5x. 

`yarn remove eslint; yarn add 'eslint@4.10.0'`

I've read that the problem comes from incompatibility w airbnb config but haven't verfied 

https://github.com/airbnb/javascript/issues/1834

**Comment by corysimmons** [comment]
If you downgrade to 4 you run into https://github.com/standard/eslint-config-standard/issues/112

**Comment by feross** [comment]
A fixed version is released as eslint-config-standard-react 7.0.0.
